content/blog/automation-api-python/index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Pulumi Automation API, Now In Python"
```

Comment:
I think the title could be more descriptive, maybe something like: "Build software automation around your infrastructure with Automation API in Python"

Reply:
I'll have to shorten that, titles have a 60 char limit.